tests/fips/fips_setup.pm

```diff
@@ -21,17 +21,22 @@ use testapi;
 use strict;
 
 # Install fips pattern and update grub.cfg to boot with fips=1
-sub run() {
+sub run {
     select_console 'root-console';
 
-    if (!script_output "grep 'fips=1' /proc/cmdline | tee") {
-        script_run 'zypper -n install --type pattern fips', 300;
```

Comment:
ah, sorry, that's my fault.
what i tried to do is to use exit code to detect 'fips=1' is enabled or not, may like this:
my $ret = script_run "grep 'fips=1' /proc/cmdline";
if ($ret ne 0)